tests/integration/conftest.py

```diff
@@ -27,4 +51,21 @@ async def _get_unit_ips(application_name: str):
             for _, unit_status in sorted(units.items(), key=lambda kv: int(kv[0].split("/")[-1]))
         )
 
-    yield _get_unit_ips
+    return get_unit_ips
+
+
+@pytest_asyncio.fixture
+async def update_config(model: Model, request: FixtureRequest, flask_app: Application):
```

Comment:
What is this fixture intended to do?

Reply:
Update the charm configuration before the test case and reset the charm configuration after the test case finished.